src/settings/settings.ts

```diff
@@ -225,9 +225,11 @@ export function getXMLSettings(javaHome: string | undefined, logfile: string, ex
   // Check workspace trust
   const isWorkspaceTrusted = (workspace as any).isTrusted;
   if (isWorkspaceTrusted !== undefined && !isWorkspaceTrusted) {
-    xml['xml']['validation']['resolveExternalEntities'] = false;
+    xml['xml']['validation']['resolveExternalEntities']['enabled'] = false;
```

Comment:
You would need to also update the settings in `package.json` and in the LemMinX settings structure.

Currently I'm seeing : `Activating extension 'redhat.vscode-xml' failed: Cannot create property 'enabled' on boolean 'false'.`

Reply:
Sorry for this mistake. The correct settings is `xml.resolveExternalEntities`. I fixedit.